sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/key_client.hpp

```diff
@@ -398,11 +403,55 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Keys {
      * @param importKeyOptions The key import configuration object containing information about
      * the #JsonWebKey being imported.
      * @param context A #Azure::Core::Context controlling the request lifetime.
+     * @return Azure::Response<KeyVaultKey>
      */
     Azure::Response<KeyVaultKey> ImportKey(
         ImportKeyOptions const& importKeyOptions,
         Azure::Core::Context const& context = Azure::Core::Context()) const;
 
+    /**
+     * @brief Lists the policy for a key.
+     *
+     * @remark The GetKeyRotationPolicy operation returns the specified key policy resources in the
+     * specified key vault. This operation requires the keys/get permission.
+     *
+     * @param name The name of the key in a given key vault.
+     * @param context A #Azure::Core::Context controlling the request lifetime.
+     * @return Azure::Response<KeyRotationPolicy>
+     */
+    Azure::Response<KeyRotationPolicy> GetKeyRotationPolicy(
+        std::string const& name,
+        Azure::Core::Context const& context = Azure::Core::Context()) const;
+
+    /**
+     * @brief Updates the rotation policy for a key.
+     *
+     * @remark Set specified members in the key policy. Leave others as undefined. This operation
+     * requires the keys/update permission.
+     *
+     * @param name The name of the key in a given key vault.
+     * @param rotationPolicy The policy for the key.
+     * @param context A #Azure::Core::Context controlling the request lifetime.
+     * @return Azure::Response<KeyRotationPolicy>
+     */
+    Azure::Response<KeyRotationPolicy> PutKeyRotationPolicy(
+        std::string const& name,
+        KeyRotationPolicy const& rotationPolicy,
+        Azure::Core::Context const& context = Azure::Core::Context()) const;
+
+    /**
+     * @brief Get the requested number of bytes containing random values.
+     *
+     * @remark Get the requested number of bytes containing random values from a managed HSM.
+     *
+     * @param options The request object to get random bytes.
+     * @param context A #Azure::Core::Context controlling the request lifetime.
+     * @return Azure::Response<std::vector<uint8_t>>
+     */
+    Azure::Response<std::vector<uint8_t>> GetRandomBytes(
```

Comment:
IIRC, we were going to wrap the vector<uint8_t> into a `GetRandomBytesResults` model.  right? @heaths

Reply:
.Net returns a byte[] , but you are correct , in C++ we wrap the byte vector in a response object